package/MDAnalysis/lib/nsgrid.pyx

```diff
@@ -375,20 +299,20 @@ cdef class NSResults(object):
         self.pair_distances2[self.npairs] = distance2
         self.npairs += 1
 
-        return self.npairs
+        return OK

```

Comment:
I checked that you never use this return code. So we can use it to unambiguously report the status of the function